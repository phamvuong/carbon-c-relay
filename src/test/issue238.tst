foo 4.2e-10 42
