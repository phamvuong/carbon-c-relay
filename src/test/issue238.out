statistics
    submit every 60 seconds
    prefix with carbon.relays.test_hostname
    ;

cluster zone1
    carbon_ch replication 1
        127.0.0.1:2101
    ;
cluster zone2
    carbon_ch replication 1
        127.0.0.1:2102
    ;
cluster zone3
    carbon_ch replication 1
        127.0.0.1:2103
    ;

match *
    validate ^[-+]?[0-9]*.?[0-9]+([eE][-+]?[0-9]+)?\ [0-9.]+$ else drop
    send to zone1
    ;
match *
    validate ^[-+]?[0-9]*.?[0-9]+([eE][-+]?[0-9]+)?\s[0-9.]+$ else drop
    send to zone2
    ;
match *
    validate ^[-+]?[0-9]*.?[0-9]+([eE][-+]?[0-9]+)?[:space:][0-9.]+$ else drop
    send to zone3
    stop
    ;

match
    * -> foo
    validate
        ^[-+]?[0-9]*.?[0-9]+([eE][-+]?[0-9]+)?\ [0-9.]+$ -> 4.2e-10 42
        match
    carbon_ch(zone1)
        127.0.0.1:2101
match
    * -> foo
    validate
        ^[-+]?[0-9]*.?[0-9]+([eE][-+]?[0-9]+)?\s[0-9.]+$ -> 4.2e-10 42
        fail -> drop
match
    * -> foo
    validate
        ^[-+]?[0-9]*.?[0-9]+([eE][-+]?[0-9]+)?[:space:][0-9.]+$ -> 4.2e-10 42
        fail -> drop
    stop
